src/shared/loop-util.c

```diff
@@ -591,11 +614,16 @@ static int loop_device_make_internal(
         if (r < 0 && r != -EOPNOTSUPP)
                 return r;
 
+        int dissect_fd = loop_device_fd_dup_for_dissect(loop_with_fd);
```

Comment:
Also here

Reply:
Thanks. I also fixed a leak in `loop_device_fd_dup_for_dissect()`.